todo-backend/README.adoc

```diff
@@ -0,0 +1,452 @@
+NAME: todo-backend
+...
+STATUS: deployed
+REVISION: 1
+----
+
+The Helm Chart for this quickstart contains all the information to build an image from the source code using Bootable Jar:
+
+[source,options="nowrap"]
+----
+build:
+  uri: https://github.com/wildfly/quickstart.git
+  mode: bootable-jar
+----
+endif::[]
+
+=== Build and Deploy the Backend on OpenShift with {productName} S2I
+
+The backend will be built and deployed on OpenShift with a Helm Chart for {productName}.
+
+ifndef::ProductRelease[]
+[source,options="nowrap"]
+----
+$ helm install todo-backend -f todo-backend-s2i.yaml wildfly/wildfly
+NAME: todo-backend
+...
+STATUS: deployed
+REVISION: 1
+----
+endif::[]
+ifdef::ProductRelease[]
+[source,options="nowrap"]
+----
+$ helm install todo-backend -f todo-backend-s2i.yaml jboss-eap/eap7
+NAME: todo-backend
+...
+STATUS: deployed
+REVISION: 1
+----
+endif::[]
+
+The Helm Chart for this quickstart contains all the information to build an image from the source code using S2I:
+
+ifndef::ProductRelease[]
+[source,options="nowrap"]
+----
+build:
+  uri: https://github.com/wildfly/quickstart.git
+  mode: s2i
+  s2i:
+    galleonLayers: cloud-server,postgresql-datasource,ejb
+----
+endif::[]
+ifdef::ProductRelease[]
+[source,options="nowrap"]
+----
+build:
+  uri: https://github.com/jboss-developer/jboss-eap-quickstarts.git
+  mode: s2i
+  s2i:
+    galleonLayers: cloud-server,postgresql-datasource,ejb
+----
+endif::[]
+
+=== Environment variables for PostgreSQL
+
+The Helm Chart also contains the environment variables required to connect to the PostgreSQL database.
+In local deployment the credentials were passed directly as the values of the environment variables.
+For OpenShift, we rely on secrets so that the credentials are never copied outside OpenShift:
+
+[source,options="nowrap"]
+----
+deploy:
+  env:
+    - name: POSTGRESQL_PASSWORD
+      valueFrom:
+        secretKeyRef:
+          key: database-password
+          name: todos-db
+----
+
+When the application is deployed, the value for the `POSTGRESQL_PASSWORD` will be taken from the key `database-password`
+in the secrets `todos-db`.
+
+Let's wait for the application to be built and deployed:
+
+[source,options="nowrap"]
+----
+$ oc get dc/todo-backend -w
+NAME           REVISION   DESIRED   CURRENT   TRIGGERED BY
+todo-backend   0          3         0         config,image(todo-backend:latest)
+...
+todo-backend   1          3         3         config,image(todo-backend:latest)
+----
+
+=== Use the todobackend Web Frontend
+
+Once the backend is deployed on OpenShift, it can be accessed from the route `todo-backend`.
+Let's find the host that we can use to connect to this backend:
+
+[source,options="nowrap"]
+----
+$ oc get route todo-backend -o jsonpath="{.spec.host}"
+todo-backend-jmesnil1-dev.apps.sandbox.x8i5.p1.openshiftapps.com
+----
+
+This value will be different for every installation of the backend.
+
+[WARNING]
+====
```

Comment:
I would add here also note that the cluster we deployed on _must_ be publicly accessible (eg. we can't test it as our test clusters are limited to company network)

In such a case, we could redirect the user to test it the same way using `curl` we do in bootable deployment (but that section will not be there for product :-1: )